packages/react/toast/src/Toast.tsx

```diff
@@ -4,7 +4,7 @@ import { composeEventHandlers } from '@radix-ui/primitive';
 import { useComposedRefs } from '@radix-ui/react-compose-refs';
 import { createContextScope } from '@radix-ui/react-context';
 import * as DismissableLayer from '@radix-ui/react-dismissable-layer';
-import { UnstablePortal } from '@radix-ui/react-portal';
+import { Portal } from '@radix-ui/react-portal';
```

Comment:
Is there a reason we alias in some places but not others?

Reply:
Yes, because a lot of the new components now expose their own `Portal` part so it would clash.

Reply:
Was just about to update my comment, should we be consistent? does it matter?

Reply:
I don't think it matter much, and generally I think we only alias when the need is there.